src/script/page/MainContent/panels/preferences/accountPreferences/DataUsageSection.tsx

```diff
@@ -60,31 +59,21 @@ const DataUsageSection: React.FC<DataUsageSectionProps> = ({
 
   useEffect(() => {
     const updateProperties = ({settings}: WebappProperties): void => {
-      setOptionPrivacy(settings.privacy.improve_wire);
       setOptionTelemetry(settings.privacy.telemetry_sharing);
     };
     amplify.subscribe(WebAppEvents.PROPERTIES.UPDATED, updateProperties);
     return () => amplify.unsubscribe(WebAppEvents.PROPERTIES.UPDATED, updateProperties);
   }, []);
 
+  const showCountly = !isTeam || (isTeam && isCountlyEnabled);
+
+  if (!showCountly && !isActivatedAccount) {
```

Comment:
Looks like it should be an `OR` at first glance, is that not the case?

Reply:
No. If both statements are falsy, the component should return null

If one of them is truthy, we want to render the component